tests/components/zwave_js/conftest.py

```diff
@@ -22,7 +22,7 @@
 @pytest.fixture(name="addon_info_side_effect")
 def addon_info_side_effect_fixture():
     """Return the add-on info side effect."""
-    return None
+    return None  # noqa: RET501
```

Comment:
Can we remove the noqa if we type the return value as `Any`?

Reply:
Yes, I can type these as Any.